README.md

```diff
@@ -15,20 +15,33 @@ interface SomeInterface {
 combined with the JavaScript implementation class file
 
 ```js
-exports.implementation = class SomeInterfaceImpl {
+module.exports = class SomeInterfaceImpl {
   add(x, y) {
     return x + y;
   }
 };
 ```
 
+> Note: It's also possible to use ES2015 module default export syntax:
```

Comment:
This is not true; jsdom only operates on CommonJS modules.

Reply:
Well, you need to transpile them first.

Reply:
Yes. That is not a supported scenario, so please remove it.